.github/workflows/release.yaml

```diff
@@ -76,18 +143,21 @@ jobs:
       run: echo "RELEASE_VERSION=$(echo ${{ github.event.milestone.title }} | cut -c 2-)" >> $GITHUB_ENV
 
     - name: Upload notes
-      uses: actions/create-release@v1
+      uses: softprops/action-gh-release@v1
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         tag_name: ${{ env.artifactId }}-${{ env.RELEASE_VERSION }}
-        release_name: Gaffer ${{ env.RELEASE_VERSION }}
+        name: Gaffer ${{ env.RELEASE_VERSION }}
         body_path: release_file.md
+        prerelease: ${{ startsWith( needs.create-release-tag.outputs.branch_name , 'alpha') }}
 
+  # Skip if alpha release
   generate-javadoc:
```

Comment:
For an alpha, this job clones master, sets some credentials, installs the JDK, sets version and then does nothing.
Instead of doing the prerequisites and skipping only the build & upload of the Javadoc, for alphas the whole job can be skipped by moving the `if` to cover the whole job and not just one step. [Described here.](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idif)

Reply:
Okay nice, I didn't know that, will amend now on all 4 PRs! I originally did it as I didn't want to keep skipping steps but didn't know you can just skip jobs!